docker/mariadb/conf.d/lower_case_table_names.cnf.macos

```diff
@@ -0,0 +1,2 @@
+[mariadb]
```

Comment:
Another pedantic comment... Would be nice to alter the readme or whatever the file is in the Docker/MariaDB folder or something like that, where it mentions the windows version of this, to just mention the macos one too.